gas.snap

```diff
@@ -0,0 +1,17 @@
+Ethereum price: $1574.08
```

Comment:
I don't think we should push this to master. I think it's great to have this info, but I don't see any benefit in pushing it

Reply:
Then I'll add it to the git ignore, does that sound good?

Reply:
Sounds good!